js/components/ProfileHeader/ProfileHeader.js

```diff
@@ -1,35 +1,51 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 import {
   ScrollView,
   View,
   Text,
   TextInput,
   Image,
   TouchableOpacity
-} from "react-native";
+} from 'react-native';
+
+import ImagePicker from 'react-native-image-crop-picker';
 
 import {
   getUserProfile,
   updateUserProfile,
   unMarshallResult
-} from "../../config/helpers";
+} from '../../config/helpers';
 
-import { styles } from "./styles";
+import { styles } from './styles';
 
-const userId = "QhP2yK3dx4P8BAB3AHJiLPAZgn93";
+const userId = 'QhP2yK3dx4P8BAB3AHJiLPAZgn93';
 
 class ProfileHeader extends Component {
   constructor(props) {
     super(props);
     this.state = {
```

Comment:
Please use Redux store to manage user info.

Reply:
I will be making a new pull request for the new feature - Redux store